index.html

```diff
@@ -2891,6 +2896,66 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
```

Comment:
I was wondering about that "store" part as well. I though it was supposed to work across stores.

We already call stores platform elsewhere like "play" - so we should reuse the same names